### PR TITLE
Add module gfx_wator

### PR DESCRIPTION
**Pull request by basxto**
Implementation of https://en.wikipedia.org/wiki/Wa-Tor

### CODE REVIEW

**Comment by vifino** [comment]
Looking good!
You can add it to the default module set too.

File: src/modules/gfx_wator.c [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+	frames = 20 * 60 * 3;//3min
+	fishs = 0;
+	sharks = 0;
+	nexttick = udate();
+	for (int i = 0; i < width * height; ++i) {
+		int rand = randn(100);
+		if (rand < 90)
+			table[i] = 0;
+		else if (rand > 98) {	//1%
+			table[i] = 2;		//shark
```
[the remark this answers is not in the document]
Comment (basxto) [reply src/modules/gfx_wator.c]: My editor aligned them with tabs

**Comment by basxto** [comment]
> Looking good!
> You can add it to the default module set too.

I just have to add it to `GFXMODS_DEFAULT` in the makefile? Or is there another place I have to add it too?

File: src/modules/gfx_wator.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (vifino) [reply src/modules/gfx_wator.c]: Spaces are a safer bet after code.
Also, please put whitespace before the `//` if its after code and always after:

DO:
```
// this is highly critical code
foo(); // foo call
```

DONT: 
```
//this is highly critical code
foo();//foo call
```

**Pull request merged**